src/main/java/org/sagebionetworks/web/client/jsinterop/ToastMessageOptions.java

```diff
@@ -141,8 +141,14 @@ public Builder setSecondaryButton(String text, AlertButtonConfig.Callback onClic
 		}
 
 		public ToastMessageOptions build() {
-			AlertButtonConfig primaryButtonConfig = AlertButtonConfig.create(primaryButtonText, onPrimaryButtonClick);
-			AlertButtonConfig secondaryButtonConfig = AlertButtonConfig.create(secondaryButtonText, onSecondaryButtonClickOrHref);
+			AlertButtonConfig primaryButtonConfig = null;
+			AlertButtonConfig secondaryButtonConfig = null;
+			if (primaryButtonText != null && onPrimaryButtonClick != null) {
+				primaryButtonConfig = AlertButtonConfig.create(primaryButtonText, onPrimaryButtonClick);
+			}
+			if (secondaryButtonText != null && onSecondaryButtonClickOrHref != null) {
+				secondaryButtonConfig = AlertButtonConfig.create(secondaryButtonText, onSecondaryButtonClickOrHref);
+			}
```

Comment:
We're still getting some empty buttons in some alerts. I think all of the alerts where this can happen are rare (in experimental mode), so IMO this doesn't need to go into staging.